<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>desencriptar</class>
 <widget class="QDialog" name="desencriptar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Desencriptar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="titulo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Nimbus Sans [UKWN]&quot;;
border: 2px solid white;</string>
   </property>
   <property name="text">
    <string>DIGITE AS CHAVES PARA DESCRIPTOGRAFAR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="continuar">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Nimbus Sans [UKWN]&quot;;
background-color: rgb(47, 47, 47);
border: 2px solid white;</string>
   </property>
   <property name="text">
    <string> CONTINUAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="e">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>q</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="n">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>p</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Nimbus Sans [UKWN]&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="e_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>e</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
